tcp_request.c

```diff
@@ -43,6 +43,8 @@ tcp_tune(evutil_socket_t handle)
         return;
     }
 
+    setsockopt(handle, IPPROTO_IP, IP_TOS, (void *) (int []) {
```

Comment:
ok thanks. maybe adding a comment just before could help the next person to understand what the intent was.